packages/eas-cli/src/build/build.ts

```diff
@@ -388,6 +409,47 @@ async function handleSingleBuildProgressAsync(
             )}`
           );
         }
+
+        if (
+          build.platform === AppPlatform.Ios &&
+          [BuildResourceClass.IosIntelLarge, BuildResourceClass.IosIntelMedium].includes(
+            build.resourceClass
+          )
+        ) {
+          let askToSwitchToM1 = false;
+          if (
+            build.priority === BuildPriority.High &&
+            build.estimatedWaitTimeLeftSeconds >= /* 10 minutes */ 10 * 60
+          ) {
+            Log.newLine();
+            Log.warn(
+              `Warning: Priority queue wait time for legacy iOS Intel workers is longer than usual (more than 10 minutes).`
+            );
+            Log.warn(`We recommend switching to the new, faster M1 workers for iOS builds.`);
+            Log.warn(`Learn more: https://blog.expo.dev/m1-workers-on-eas-build-dcaa2c1333ad`);
```

Comment:
```suggestion
            Log.warn(learnMore('https://blog.expo.dev/m1-workers-on-eas-build-dcaa2c1333ad', {
              learnMoreMessage: 'See more information about M1 workers.',
            });
```
Or something like this, feel free to adjust message